cartographer/io/ply_writing_points_processor.cc

```diff
@@ -61,6 +64,11 @@ void WriteBinaryPlyPointCoordinate(const Eigen::Vector3f& point,
   CHECK(file_writer->Write(buffer, 12));
 }
 
+void WriteBinaryIntensity(const float intensity,
+                          FileWriter* const file_writer) {
+  CHECK(file_writer->Write(&intensity, sizeof(float)));
```

Comment:
Does this compile? It looks like it is missing a `reinterpret_cast<const char*>`.

Reply:
does now.